src/migrations/1618911977800-lifecycle-application.ts

```diff
@@ -0,0 +1,21 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```

Comment:
I suggest we merge all 4 migrations into one.

Reply:
Agreed, done